docker/files/etc/nginx/conf.d/default.conf

```diff
@@ -4,12 +4,39 @@ server {
     server_name localhost;
     charset utf-8;
 
-    location / {
+    location @proxy {
         proxy_pass http://app-production:8000;
         proxy_set_header Host $host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto $scheme;
     }
 
+    location / {
+        rewrite ^(.*)$ /static/frontend/index.html last;
+    }
+
+    location /index.html {
+        try_files $uri @proxy;
+    }
+
+    # Frontend static files (file with any extension)
+    location ~ ^/assets {
```

Comment:
I'm not sure to understand this rule and why you need it. Is it related with how the front application is using static in its code ?

Reply:
the front application looks for its static files in `/assets` but the frontend assets are stored in the backend static folder in `/static/frontend/` to be served by whitenoise.

Reply:
Ok but doing this how the front is using the CDN to use the static ? Where this information is stored and how it is used ?

Reply:
Yes but the front application must be aware of the CDN url. Otherwise it will fetch the application url and never the CDN one.

If I write in my code `<img src="/assets/img/dummy.png" />` then in your inspector you will see a request made on `https://jitsi-magnify.com/assets/img/dummy.png` and not on the CDN.
I f the application use the webpack split feature, webpack must be aware of the CDN url, otherwise each chunk will be fetch on the application URL and not on the CDN.

Reply:
Ah. But the application itself is hosted behind the CDN.
But anyway we are planning a manifest.json at the root of the project that the frontend will get on startup to know the url of Keycloak... and can also get the url of the CDN if necessary.

Reply:
The CDN configuration is made in the terraform of the project ? To see how it is handle.